Clients/src/presentation/pages/Assessment/index.tsx

```diff
@@ -1,5 +1,72 @@
+import Button from "@mui/material/Button";
+import Stack from "@mui/material/Stack";
+import { Paper, styled, Typography, useTheme } from "@mui/material";
+
+const Item = styled(Paper)(({ theme }) => ({
```

Comment:
For the next step
Please move these stylings to their components as in-line styles